csrankings.csv

```diff
@@ -15199,6 +15199,7 @@ Xiaoli Z. Fern,Oregon State University,http://web.engr.oregonstate.edu/~xfern,rn
 Xiaoli Zhang Fern,Oregon State University,http://web.engr.oregonstate.edu/~xfern,rnDD_oEAAAAJ
 Xiaolin Hu,Tsinghua University,http://www.xlhu.cn,PksdgoUAAAAJ
 Xiaolin Hu 0002,Georgia State University,https://grid.cs.gsu.edu/xhu,zeU19nAAAAAJ
+Xiaolin Xu,University of Illinois at Chicago,https://xiaolin8.people.uic.edu/,U6TRlVgAAAAJ

```

Comment:
Not listed as faculty in CS or adjunct faculty.
https://cs.uic.edu/faculty-staff/adjunct-faculty/2/
This means that he does not meet the inclusion criteria.